src/rbdmap

```diff
@@ -58,19 +58,21 @@ do_map() {
 
 unmount_unmap() {
 	local rbd_dev=$1
-	local mnt=$(findmnt --mtab --source ${rbd_dev} --noheadings \
+	local mnts=$(findmnt --mtab --source ${rbd_dev} --noheadings \
 							| awk '{print $1'})
 
 	logger -p "daemon.debug" -t rbdmap "Unmapping '${rbd_dev}'"
-	if [ -n "${mnt}" ]; then
-	    logger -p "daemon.debug" -t rbdmap "Unmounting '${mnt}'"
-	    umount "${mnt}" >>/dev/null 2>&1
-	fi
-	if mountpoint -q "${mnt}"; then
-	    ## Un-mounting failed.
-	    logger -p "daemon.warning" -t rbdmap "Failed to unmount '${mnt}'"
-	    return 1
-	fi
+	for mnt in ${mnts}; do
+	    if [ -n "${mnt}" ]; then
```

Comment:
I think this "if" is pointless now; if ${mnts} is empty we won't even get here

Reply:
Right, will remove the if